src/CLI/actioner/sonic-cli-neighbors.py

```diff
@@ -211,8 +211,12 @@ def run(func, args):
         elif 'sonic-neighbor:output' in response.keys():
             status = response['sonic-neighbor:output']
             status = status['response']
-            if (status != "Success"):
+            if "255" in status:
```

Comment:
We need the below message for REST intf as well, please add appropriate error/warn string in the RPC handler.

Reply:
I know, this is a temporary fix as any major change will not be merged right now. Post Buzznik there would be a major update that will address all these issues.